AUTHORS

```diff
@@ -3507,5 +3507,9 @@ BuShe <bushe.cn@icloud.com>
 Nick Sia <31839263+nicksia-vgw@users.noreply.github.com>
 Basit Chonka <baasit121@icloud.com>
 Jeremiah Gowdy <jgowdy@godaddy.com>
+Nick Sia <nicholas.sia@vgw.co>
```

Comment:
This one's a duplicate (just three lines above) and we'll need to add a .mailmap entry for it.